models/users-models.js

```diff
@@ -1,2 +1,7 @@
 const db = require("../db/connection")
 
+exports.getAllUsers = () => {
+    return db.query(`
+    SELECT * FROM users;
+    `)
+}
```

Comment:
It would be better to handle as much data in the model as possible rather than returning the whole query string to the controller and handling it there 💭